src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/MainPageTests.java

```diff
@@ -153,7 +153,7 @@ public void MercuryCuratedMainPageTest_001_CheckElementsVisibilityElementsOrderA
     //TODO: make function from this
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)